### DISCUSSION TITLE
Any single field in a form is required?

### DISCUSSION THREAD

**Discussion by seanmthompson** [react-hook-form/react-hook-form#4136] [Q&A] [answered]
Is there a way to have a group of inputs, and only one of them needs to have value for the form to be valid? So say, I have 5 text fields, and just one needs to have a value for the form to be valid, but if none have input, the form isn't valid.. is this possible?

**Comment by bluebill1049** [comment, marked as answer]
you should be able to this in the following options

- `validate` function by using getValues and return a function
- schema validation
- build a controlled component and wrap with Controller and write a validate function.
